bonito/openvino/loader.py

```diff
@@ -0,0 +1,99 @@
+# This script provides a method which builds OpenVINO network in runtime
+import numpy as np
+from openvino.inference_engine import IECore, IENetwork
+
+import ngraph as ng
+from ngraph.impl.op import Parameter
+from ngraph.impl import Function, Shape, Type
+
+import torch
+from torch.autograd import Variable
+
+
+nodes = {}
+out = None
+
+def forward_hook(self, inputs, output):
+    global out
+    layer_type = self.__class__.__name__
+
+    params = [value.numpy() for value in self.state_dict().values()]
+
+    inp = nodes[inputs[0].data_ptr()]
+    if layer_type == 'Conv1d':
+        weights = np.expand_dims(params[0], axis=2)
+        if self.groups == 1:
+            out = ng.convolution(inp, weights,
```

Comment:
Scalability issue: ideally we should have a pluggable table of 1: 1 and 1:many connections between the torch and openvino opsets.